Curve-finance-suite/Pool-Migration/src/agent.spec.ts

```diff
@@ -0,0 +1,199 @@
+import {
+  Finding,
+  HandleTransaction,
+  FindingSeverity,
+  FindingType,
+  TransactionEvent
+} from 'forta-agent';
+import { provideMigratePoolAgent } from './agent';
+import {
+  createAddress,
+  TestTransactionEvent,
+  encodeFunctionCall
+} from "forta-agent-tools";
+
+const ADDRESS = createAddress('0x1111');
+const ALERT_ID = 'test';
+
+describe('Pool Migration Agent', () => {
+  let handleTransactions: HandleTransaction;
+
+  const _old_pool = createAddress('0x1');
+  const _new_pool = createAddress('0x2');
+  const _amount = '1000'
+  const _from = createAddress('0x3');
+
+  const _input: string = encodeFunctionCall(
+    {
+      name: 'migrate_to_new_pool',
+      type: 'function',
+      inputs: [
+        {
+          type: 'address',
+          name: '_old_pool',
+        },
+        {
+          type: 'address',
+          name: '_new_pool',
+        },
+        {
+          type: 'uint256',
+          name: '_amount',
+        },
+      ],
+    },
+    [_old_pool, _new_pool, _amount]
```

Comment:
I think you could change this to use `PM_IFACE` something like `PM_IFACE.encodeFunctionData("migrate_to_new_pool", [_old_pool, _new_pool, _amount]`

Reply:
Looked up the `ethers` documentation after your comment. Wow. That's great. Already updated.